python/GafferUI/SplinePlugValueWidget.py

```diff
@@ -75,12 +75,15 @@ def setHighlighted( self, highlighted ) :
 
 		self.splineWidget().setHighlighted( highlighted )
 
-	def _updateFromPlug( self ) :
+	def _updateFromValues( self, values, exception ) :
 
-		plug = self.getPlug()
-		if plug is not None :
-			with self.getContext() :
-				self.__splineWidget.setSpline( plug.getValue().spline() )
+		if values :
+			assert( len( values ) == 1 )
+			self.__splineWidget.setSpline( values[0].spline() )
+		else :
+			self.__splineWidget.setSpline(
+				IECore.Splineff( IECore.CubicBasisf.linear(), [ ( 0, 0.3 ), ( 1, 0.3 ) ] ),
```

Comment:
Why a default value of 0.3?

Reply:
It makes a neutral non-gradated spline, which serves as a reasonable visual indication of "value not available".